apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,788 @@
+              </v-tooltip>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </div>
+    </template>
+    <template #main-content>
+      <v-container fluid grid-list-md pt-0 mt-4 mx-1>
+        <v-row>
+          <v-col md="4" :cols="12" class="pr-0 pl-1">
+            <!-- Data Table -->
+            <v-card height="94vh" overflow-auto>
+              <v-card-title class="pt-2">
+                <div
+                  style="
+                    display: flex;
+                    align-items: center;
+                    justify-content: space-between;
+                    width: 100%;
+                  "
+                >
+                  <div>
+                    <strong
+                      >Rules ({{ numItems }} shown,
+                      {{ loadedRules.length - numItems }} hidden)</strong
+                    >
+                  </div>
+                  <div style="width: fit-content">
+                    <v-switch
+                      v-model="shortIdEnabled"
+                      style="margin-top: 0; padding-top: 0"
+                      dense
+                      inset
+                      label="Short ID"
+                      color="teal"
+                      hide-details
+                    />
+                  </div>
+                </div>
+                <v-spacer class="mt-0 pt-0" />
+                <v-select
+                  v-model="selectedHeaders"
+                  :items="headersList"
+                  label="Select Columns"
+                  class="mt-4 pt-0"
+                  dark
+                  item-color="white"
+                  multiple
+                  outlined
+                  return-object
+                  height="8.5vh"
+                >
+                  <template #selection="{item, index}">
+                    <div v-if="$vuetify.breakpoint.lgAndUp">
+                      <v-chip v-if="index < 4" small>
+                        <span>{{ item.text }}</span>
+                      </v-chip>
+                      <span
+                        v-if="index === 4"
+                        class="grey--text caption mt-0 pt-0"
+                        >(+{{ selectedHeaders.length - 4 }} others)</span
+                      >
+                    </div>
+                    <div v-else>
+                      <v-chip v-if="index < 3" small>
+                        <span>{{ item.text }}</span>
+                      </v-chip>
+                      <span v-if="index === 3" class="grey--text caption ml-2"
+                        >(+{{ selectedHeaders.length - 3 }} others)</span
+                      >
+                    </div>
+                  </template>
+                </v-select>
+              </v-card-title>
+              <v-card-text>
+                <v-data-table
+                  :single-select="true"
+                  disable-pagination
+                  dense
+                  mobile-breakpoint="0"
```

Comment:
please add comment explaining why this potentially resolves the issue where the table completely breaks when you make the screen too narrow

Reply:
can you also add this fix to the other places where we use v-data-table and could run into this width issue?